cucumber/authenticators_gce/features/step_definitions/authn_gce_steps.rb

```diff
@@ -14,19 +15,44 @@
     raise "Incorrect annotation name: '#{annotation_name}', expected: service-account-id|service-account-email|project-id|instance-name"
   end
 
-  set_annotation_to_resource("#{annotation_name}", annotation_value)
+  set_annotation_to_resource("authn-gce/#{annotation_name}", annotation_value)
 end
 
-Given(/^I obtain a GCE identity token in "([^"]*)" format with audience claim value: "([^"]*)"$/) do | audience, format |
+# Sets all GCE annotations
+Given(/^I set GCE annotations to host "([^"]*)"$/) do | hostname |
```

Comment:
why aren't you using this in the basic vanilla test?

Reply:
Wanted to show the full set.